workers/jobs/google_scraper.go

```diff
@@ -80,14 +79,14 @@ func (c *Context) PerformSearchJob(job *work.Job) error {
 
 // Update status to failed when (jobFails + 1) reach MaxFails. Note: Job won't retry if jobFails reach MaxFails
 // So this need to be done at jobFails + 1
-func updateStatusToFailed(jobFails int64, jobName string, keywordID uint, keyword string, err error) {
+func updateStatusToFailed(jobFails int64, jobName string, keywordID uint, keyword string, failedReason error) {
 	if int(jobFails)+1 >= MaxFails {
-		updateStatusErr := google_search_service.UpdateKeywordStatus(keywordID, models.Failed, err)
+		err := google_search_service.UpdateKeywordStatus(keywordID, models.Failed, failedReason)
 
-		if updateStatusErr != nil {
-			panic(fmt.Sprintf("Cannot update keyword status (reason: %v)", updateStatusErr))
+		if err != nil {
+			log.Fatal("Failed to update keyword status: ", err)
```

Comment:
@gutakk Should we add this as a constant in `helpers/error_handler/error_constant.go`?

Reply:
Same as https://github.com/gutakk/go-google-scraper/pull/78#discussion_r573613521